gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,10 +1,10 @@
 #
-# Copyright (c) VMware, Inc. 2022-2023. All rights reserved.
+# Copyright 2022-2024 Broadcom. All rights reserved.
```

Comment:
We shouldn't be committing this file, right?

Reply:
you are correct... 😮‍💨